src/main/scala/edu/cmu/cs/obsidian/codegen/CodeGen.scala

```diff
@@ -1191,7 +1191,7 @@ class CodeGen (val target: Target) {
 
         def handleNonPrimitive(name: String, n: ObsidianType): Unit = {
             // foo = new Foo(); foo.initFromArchive(archive.getFoo());
-            val javaFieldTypeName = javaFieldType.fullName()
+            // val javaFieldTypeName = javaFieldType.fullName()
```

Comment:
Please delete this line and the commented-out line above it.

Reply:
I think the commented-out line above it is to say what code will be generated from jCodeModel. But, yeah, I'll delete the other one -- it was flagged as an unused declaration.